src/ZendeskApi.Contracts/Models/Ticket.cs

```diff
@@ -1,114 +1,110 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel;
-using System.Runtime.Serialization;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Converters;
 
 namespace ZendeskApi.Contracts.Models
 {
     [Description("Ticket")]
-    [DataContract]
     public class Ticket : IZendeskEntity
     {
-        [DataMember(EmitDefaultValue = false)]
+        [JsonProperty]
         public long? Id { get; set; }
 
-        [DataMember(Name = "created_at", EmitDefaultValue = false)]
+        [JsonProperty("created_at")]
         public DateTime? Created { get; set; }
 
-        [DataMember(Name = "updated_at", EmitDefaultValue = false)]
+        [JsonProperty("updated_at")]
         public DateTime? Updated { get; set; }
 
-        [DataMember(Name = "due_at", EmitDefaultValue = false)]
+        [JsonProperty("due_at")]
         public DateTime? Due { get; set; }
 
         [JsonConverter(typeof(StringEnumConverter))]
-        [DataMember(Name = "type", EmitDefaultValue = false)]
+        [JsonProperty("type")]
         public TicketType? Type { get; set; }
 
-        [DataMember(Name = "subject")]
+        [JsonProperty("subject")]
         public string Subject { get; set; }
 
-        [DataMember(Name = "description")]
+        [JsonProperty("description")]
         public string Description { get; set; }
 
-        [DataMember(Name = "comment")]
+        [JsonProperty("comment")]
         public TicketComment Comment { get; set; }
 
         [JsonConverter(typeof(StringEnumConverter))]
-        [DataMember(Name = "status", EmitDefaultValue = false)]
+        [JsonProperty("status")]
         public TicketStatus Status { get; set; }
 
-        [DataMember(Name = "requester_id", EmitDefaultValue = false)]
+        [JsonProperty("requester_id")]
         public long? RequesterId { get; set; }
 
-        [DataMember(Name = "submitter_id", EmitDefaultValue = false)]
+        [JsonProperty("submitter_id")]
         public long? SubmitterId { get; set; }
 
-        [DataMember(Name = "assignee_id", EmitDefaultValue = false)]
+        [JsonProperty("assignee_id")]
         public long? AssigneeId { get; set; }
 
-        [DataMember(Name = "organization_id", EmitDefaultValue = false)]
+        [JsonProperty("organization_id")]
         public long? OrganisationId { get; set; }
 
-        [DataMember(EmitDefaultValue = false)]
+        [JsonProperty()]
         public Uri Url { get; set; }
 
-        [DataMember(Name = "priority", EmitDefaultValue = false)]
+        [JsonProperty("priority")]
         public string Priority { get; set; }
 
-        [DataMember(Name = "recipient", EmitDefaultValue = false)]
+        [JsonProperty("recipient")]
         public object Recipient { get; set; }
 
-        [DataMember(Name = "group_id", EmitDefaultValue = false)]
+        [JsonProperty("group_id")]
         public int? GroupId { get; set; }
```

Comment:
as per #73 we should get this changed to long, i can do this in a seperate PR though.